src/app/training/training.component.html

```diff
@@ -0,0 +1 @@
+<p>training works!</p>
```

Comment:
training component is not part of loginAndRegister feature. Should be commited to different branch.